go.mod

```diff
@@ -5,6 +5,7 @@ go 1.15
 require (
 	github.com/go-logr/logr v0.3.0
 	github.com/go-logr/zapr v0.3.0 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
where is the replace directive that fix the version?

Reply:
yes, I updated the commit.  

Previously, I am using the `replace` , but with the latest push,  I changed to use indirect require.